src/lib/components/Codelab/index.js

```diff
@@ -84,21 +86,14 @@ class Codelab extends BaseElement {
     if (!this.glitch) {
       return html``;
     }
-    const isTest = env === 'test';
-
-    // If this is a test, always show the warning. Percy snapshots our DOM at a
-    // low resolution before resizing it, so we can't rely on _isDesktop being
-    // different for smaller or larger tests. The `w-test` ensures we test the
-    // sticky behavior of this element.
-    if (!this._isDesktop || isTest) {
-      const message = isTest
-        ? `This Glitch isn't loaded in a test environment`
-        : `This Glitch isn't available on small screens`;
+
+    if (!this._isDesktop) {
```

Comment:
Is the warning written in lines 89-92 not relevant anymore?

Reply:
I think so, it's to hide the codelab when on mobile (as the viewport is too small)